featured.js

```diff
@@ -0,0 +1,143 @@
+/* eslint-disable linebreak-style */
+
+const speakerCards = [
+  {
+    card: 1,
+    name: 'Barr Moses',
+    image: './images/speaker1.jpg',
+    description: 'Co-founder & CEO at Monte Carlo.',
+    description2: `Barr is co-founder of data reliability company Monte Carlo.
+      Barr is co-founder of data reliability company Monte Carlo.`,
+  },
+  {
+    card: 2,
+    name: 'Changpeng Zhao',
+    image: './images/speaker2.png',
+    description: 'Co-founder & CEO at Binance',
+    description2: `In 2020, Bloomberg recognised Changpeng as one of the
+      most influential people of the year, and as someone who has changed the global business landscape. `,
+  },
```

Comment:
- [ ] Nice work making the changes suggested by the previous reviewer in displaying the cards. however, there's a slight margin between the image and the text which you need to make. This gives a better user experience based on the figma design 👍 

## Your view:
![image](https://user-images.githubusercontent.com/50990925/219526442-6e166bc4-66b1-498a-82d0-934dddba1eed.png)

## Figma
![image](https://user-images.githubusercontent.com/50990925/219526496-e8a9040d-c4a6-4fe9-b1f3-cea798b96f54.png)

Reply:
Suggested changes have been incorporated within the project (margin added between image and text). Screen shot attached below for your kind reference.
![image](https://user-images.githubusercontent.com/117091923/219598999-4bee86a0-f2da-4e5c-acb1-6c4602c3f178.png)
![image](https://user-images.githubusercontent.com/117091923/219599646-f1478df7-170f-4123-ab2d-988d859b26d1.png)